package.json

```diff
@@ -24,7 +24,7 @@
   "peerDependencies": {
     "code-push": ">=1.8.0-beta",
     "cordova-plugin-file-transfer": ">=1.3.0",
-    "cordova-plugin-file": ">=3.0.0",
+    "cordova-plugin-file": "^3.0.0",
```

Comment:
if we're going to fix the cordova dependency in place to get around the file plugin bug, then I suggest fixing it to the commit that fixes the marshmallow permissions issue and verifying that it was before the bug that breaks the plugin. otherwise, my automated tests will not work.